bootstrapvz/providers/ec2/tasks/packages.py

```diff
@@ -9,9 +10,22 @@ class DefaultPackages(Task):
 	@classmethod
 	def run(cls, info):
 		info.packages.add('file')  # Needed for the init scripts
-		import os.path
+
 		kernel_packages_path = os.path.join(os.path.dirname(__file__), 'packages-kernels.yml')
 		from bootstrapvz.common.tools import config_get
 		kernel_package = config_get(kernel_packages_path, [info.manifest.release.codename,
 		                                                   info.manifest.system['architecture']])
 		info.packages.add(kernel_package)
+
+
+class AddWorkaroundGrowpart(Task):
+	description = 'Adding growpart'
```

Comment:
Maybe the description should be a little more explanatory. i.e. "Adding growpart workaround for jessie"